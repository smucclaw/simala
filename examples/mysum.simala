let opaque mysum = fun (xs) => foldl(fun (acc, x) => acc + x, 0, xs)
in  mysum([1,2,3,4])
