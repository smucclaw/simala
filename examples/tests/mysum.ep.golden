opaque mysum = fun (xs) => foldl(fun (acc, x) => acc + x, 0, xs) ;
#eval mysum([1,2,3,4])
